hsds/datanode_lib.py

```diff
@@ -829,6 +956,7 @@ async def get_chunk(
 
     # validate arguments
     if s3path:
+        """
```

Comment:
Why remove this validation?

Reply:
Looks like this is not relevant after latest code changes.